pkg/network/event_common.go

```diff
@@ -77,6 +77,31 @@ func (d ConnectionDirection) String() string {
 	}
 }
 
+// EphemeralPortType will be either EphemeralUnknown, EphemeralTrue, EphemeralFalse
+type EphemeralPortType uint8
+
+const (
+	// EphemeralUnknown indicates inability to determine whether the port is in the ephemeral range or not
+	EphemeralUnknown EphemeralPortType = 0
```

Comment:
nit: May be cleaner to use [`iota`](https://medium.com/swlh/iota-create-effective-constants-in-golang-b399f94aac31) instead of hardcoded consts here